azblob/zz_response_helpers.go

```diff
@@ -67,7 +67,7 @@ func (r *DownloadResponse) Body(o RetryReaderOptions) io.ReadCloser {
 				BlobAccessConditions{
 					ModifiedAccessConditions: ModifiedAccessConditions{IfMatch: getInfo.ETag},
 				},
-				false)
+				false, encKey)
```

Comment:
This argument was missing

Reply:
Should we have one arg per line or is that convention not followed in the SDK?

Reply:
It's not really followed in the SDK :) e.g. line 66 above